content/accessibility/links.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Links
+---
+
+## Overview
+Links are user interface elements that *navigate* you to a new place or new content. Contrast this with buttons, which are designed to *activate* a feature.
+
+## Why?
+Links can do things like help with page context, reference similar items of interest, and allow for endless connected information surfing through sites like Wikipedia. Links play a key part in your experience on the web, but without proper consideration they can be frustrating to use, skipped over, or completely unnoticed. 
+
+For screen reader assistive technology, links and buttons are expected to function differently from each other. If a link is activated and does not do what was expected, that can be disorienting and frustrating. 
```

Comment:
I wonder if it would be helpful to provide a video example to make this even more clear.

Video idea: using the rotor, navigating through the page links

Reply:
I took a video of the rotor experience with links and the buttons to show their differences in expectations but I'm having trouble getting it to display on the doc site properly. I don't know that our docs site is set up for videos? I'm reluctant to put a GIF on there since people can't stop it. If anyone knows how to get a video working on docs site or if it's allow I'm happy to know!

In lieu of a video I can always take screen shots of the link list and button list to help provide context. Thoughts?